### PR TITLE
rename to follow name convention

### PR DESCRIPTION
**Pull request by silabs-krdosvik**
Follow naming convention.

I dont think this will pass formal or ci check without the upcomming changes on cv32e40x-dv.

### CODE REVIEW

**Review by silabs-robin — CHANGES_REQUESTED** [review]
Blocking based on:
_"I dont think this will pass formal or ci check without the upcomming changes on cv32e40x-dv."_.
We had decided that `cv32e40x/dev` should always be clean.

File: lib/uvm_agents/uvma_rvfi/uvma_rvfi_instr_if.sv
Comment (silabs-robin) [inline lib/uvm_agents/uvma_rvfi/uvma_rvfi_instr_if.sv:1]: https://github.com/openhwgroup/cv32e40x-dv/pull/39#discussion_r1325451715

Comment (silabs-robin) [inline lib/uvm_agents/uvma_rvfi/uvma_rvfi_instr_if.sv:1]: > I dont think this will pass formal or ci check without the upcomming changes on cv32e40x-dv.

_"the upcomming changes on cv32e40x-dv"_ refers to https://github.com/openhwgroup/cv32e40x-dv/pull/39 , right?

That one is merged now, but it will have zero effect here until the hash is updated too.

Comment (silabs-krdosvik) [reply lib/uvm_agents/uvma_rvfi/uvma_rvfi_instr_if.sv:1]: Correct. Latest commit fix this.

Comment (silabs-krdosvik) [reply lib/uvm_agents/uvma_rvfi/uvma_rvfi_instr_if.sv:1]: NM! What is needed is an update of the hash in the cv32e40x-dv repo: https://github.com/openhwgroup/cv32e40x-dv/pull/48. The cv32e40x-dv PR can be merged after this is merged. But should be merged pretty quickly afterwards.

Comment (silabs-robin) [reply lib/uvm_agents/uvma_rvfi/uvma_rvfi_instr_if.sv:1]: The hash could be updated from this PR.
But I can also just quickly merge both in succession.
Am I guaranteed that ci_check passes after merging both?

**Comment by silabs-krdosvik** [comment]
Now, https://github.com/openhwgroup/cv32e40x-dv/pull/48 is merged. This can than be merged.

File: bin/clonetb
Comment (silabs-robin) [inline bin/clonetb:1]: Is ci_check passing and formal working now that all other PRs are merged and the hash to x-dv is updated?

Comment (silabs-krdosvik) [reply bin/clonetb:1]: Everything listed ^, seems to work fine, except clic test on ci check

Comment (silabs-robin) [reply bin/clonetb:1]: Awesome. Thanks.

**Review by silabs-robin — APPROVED** [review]
Approving based on:
All conversations resolved.
Test status reportedly ok.

**Pull request merged**
